extensions/2.0/Khronos/KHR_materials_iridescence/schema/glTF.KHR_materials_iridescence.schema.json

```diff
@@ -0,0 +1,50 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title": "KHR_materials_iridescence glTF extension",
+    "type": "object",
+    "description": "glTF extension that defines an iridescence effect.",
+    "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
+    "properties": {
+        "iridescenceFactor": {
+            "type": "number",
+            "description": "The iridescence intensity factor.",
+            "default": 0.0,
+            "minimum": 0.0,
+            "maximum": 1.0,
+            "gltf_detailedDescription": ""
```

Comment:
Three parameters have a blank value for `gltf_detailedDescription`.  These should be either removed or populated.